src/components/ProtectedRoute/ProtectedRoute.tsx

```diff
@@ -1,8 +1,8 @@
 import { Navigate, Outlet } from 'react-router-dom';
 
 function ProtectedRoute() {
-    const isLogged = localStorage.getItem('isLogged');
-    const isUser = localStorage.getItem('user');
+    const isLogged = localStorage.getItem('refreshToken');
+    const isUser = localStorage.getItem('isAdmin') == 'user';
```

Comment:
let's rename it to either role or modify it to be true / false

Reply:
ok. thanks for suggestion. I will change it